src/Aeon.Acquisition/SynchronizerMonitor.bonsai

```diff
@@ -91,13 +91,20 @@
       <Expression xsi:type="MemberSelector">
         <Selector>Item2</Selector>
       </Expression>
+      <Expression xsi:type="harp:Parse">
+        <harp:Register xsi:type="harp:ParseMessagePayload">
+          <harp:PayloadType>Timestamp</harp:PayloadType>
+          <harp:IsArray>false</harp:IsArray>
+        </harp:Register>
+      </Expression>
       <Expression xsi:type="Combinator">
         <Combinator xsi:type="rx:Zip" />
```

Comment:
Just to confirm, since I haven't the visual workflow layout in mind, this Zip will is guaranteed to receive a single input from each source on each sync trigger, is that right?

Reply:
Yes, just confirmed.
The Zip has 3 inputs:

1. The first two sources are a buffer trigger from the third one, so an array even if empty.
2. This array is converted to a list with a merge so a single element even if an empty list for each.
3. The third one a Timestamp constructed based on the Harp heart bit  that triggered the first two.